app/src/main/java/com/example/dz/MyVM2.kt

```diff
@@ -0,0 +1,113 @@
+package com.example.k_02
+
+import android.widget.Toast
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.example.dz.MainActivity6
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.launch
+import kotlin.random.Random
+
+
+class MyVM2 : ViewModel() {
+
+    var potatoes1 = MutableLiveData(0)
```

Comment:
Create class to keep potatoes value,cabbage value and so on

Reply:
next time I'll do better)) 
this time the customer is state and does not demand more)

Reply:
=))